.github/actions/paths-filter/action.yml

```diff
@@ -43,10 +55,18 @@ runs:
         java-code-change:
           - '**/src/main/**'
           - '**/src/test/**'
-          
+
         maven-change:
           - 'bom/*'
           - 'build-tools/**'
           - 'dist/**'
           - 'parent/*'
           - 'pom.xml'
+
+        docker-change:
+          - 'Dockerfile'
+          - '*.Dockerfile'
+
+        frontend-change:
```

Comment:
🔧 I guess we want to run also on identity frontend change? Or do we skip it for now as it is not part of the single app?

Reply:
Might be missed later to update this here

Reply:
it is actually included in `docker-ci` path filter https://github.com/camunda/camunda/blob/3ea015fb0229b32b70fa569c1c52524f9417dc3a/.github/actions/paths-filter/action.yml#L40-L41
but you are right, I will include it here in this general `frontend-change` filter

Reply:
fixed in https://github.com/camunda/camunda/pull/19623/commits/ecd6e586376a0d2feedd0bc87cf4fbc31594a4b8